### PR TITLE
simple notepad

### LINKED ISSUE #302: Simple Notepad
# Aim

To provide a very simple GUI based notepad

# Details

It will have saving text file and opening and basic editing features.

### PR DESCRIPTION
**Pull request by Bug-007**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #302 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by atarax665 — COMMENTED** [review]
Add proper README according to [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md) template.

**Comment by Bug-007** [comment]
I have created README.md File and have included instructions.

File: Simple Notepad/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Package/Script Name
+
+Using Tkinter GUI of simplenotepad is made.
+
+## Setup instructions
+
+Just run script and you are good to go.
+
+## Output
+![simplenotepad](https://cdn1.bbcode0.com/uploads/2021/1/28/8c8c0aad51d75a5b6cb9db6a2a775118-full.png)
```
Comment (atarax665) [inline Simple Notepad/README.md:10]: Add link of the image that you have added.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Simple Notepad/README.md (hunk 1 above)
Comment (Bug-007) [reply Simple Notepad/README.md]: Okay , I have fixed it now please check it.

**Comment by Bug-007** [comment]
I have fixed that problem please, review it.

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚀

**Comment by avinashkranjan** [comment]
@all-contributors please add @Bug-007  for code and documentation

**Pull request merged**